spring-cloud-gcp-samples/spring-cloud-gcp-vision-api-sample/src/main/java/com/example/VisionController.java

```diff
@@ -16,6 +16,7 @@
 
 package com.example;
 
+import java.io.IOException;
```

Comment:
This import is out of use. Not sure why it doesn't fail for you with Checkstyle...